gemrb/core/Map.cpp

```diff
@@ -3679,9 +3677,7 @@ void AreaAnimation::InitAnimation()
 void AreaAnimation::SetPalette(ieResRef Pal)
 {
 	Flags |= A_ANI_PALETTE;
-	gamedata->FreePalette(palette, PaletteRef);
-	strnlwrcpy(PaletteRef, Pal, 8);
-	palette = gamedata->GetPalette(PaletteRef);
```

Comment:
PaletteRef is a member variable, so this would make it go out of sync.

Best to also keep strnlwrcpy in, since we'll have to remove it from ResRef if we'll want to use it more often. Changing case of stored strings have savegame compatibility implications, so it's better to be safe.

Reply:
This is fixed in the latest push.